### DISCUSSION TITLE
컬럼 우측에 있는 아이콘 사이즈 조절 문의

### DISCUSSION THREAD

**Discussion by lunapsy** [bosskmk/pluto_grid#546] [Q&A]
모바일폰에서는 컬럼 아이콘이 작아서 그런지 잘 눌러지지가 않습니다.

다른 방법이 있는지 문의드립니다

**Comment by bosskmk** [comment]
사용하시는 모바일의 해상도가 HDPI 인가요?

https://github.com/bosskmk/pluto_grid/issues/518#issuecomment-1198656211

위 링크와 동일한 문제인거 같은데 링크에서 pubspec.yaml 파일 의 dependencies 을 변경 하여 테스트 해보실 수 있나요?
